### PR TITLE
July 26 - August 31

### PR DESCRIPTION
**Pull request by bahriaditi**
Latest design updates from the team (Vicky, Kenny, Keeth, Aditi)

### CODE REVIEW

File: aries-site/src/pages/whats-new.mdx [hunk 1]
```diff
@@ -51,9 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## July 26 - August 31, 2021 
+- Published the [Notification](https://design-system.hpe.design/components#visualizations) and [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications) & [Status Indicator](https://design-system.hpe.design/templates/status-indicator) content with examples.
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:55]:
```suggestion
- Published the [Notification](/components/notification) and [Toast Notifications](/templates/toast-notifications) & [Status Indicator](/templates/status-indicator) content with examples. 
```

**Review by britt6612 — APPROVED** [review]

File: aries-site/src/pages/whats-new.mdx [hunk 2]
```diff
@@ -51,9 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## July 26 - August 31, 2021 
+- Published the [Notification](https://design-system.hpe.design/components#visualizations) and [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications) & [Status Indicator](https://design-system.hpe.design/templates/status-indicator) content with examples. 
+- Updated [toolbar button to include an icon only variant + changed stroke from 2px to 1px](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1)
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:56]: Capitalization consistency between "Toolbar Button" and "Default Button" listed below.

End sentences with period.

Trying to not hyperlink the majority of the sentence, focussing on the subject.

```suggestion
- Updated [Toolbar Button](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1)  to include an icon only variant + changed stroke from 2px to 1px.
```

File: aries-site/src/pages/whats-new.mdx [hunk 3]
```diff
@@ -51,9 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## July 26 - August 31, 2021 
+- Published the [Notification](https://design-system.hpe.design/components#visualizations) and [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications) & [Status Indicator](https://design-system.hpe.design/templates/status-indicator) content with examples. 
+- Updated [toolbar button to include an icon only variant + changed stroke from 2px to 1px](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1) 
+- Updated [Default Button](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=681%3A5164) in Figma library to align with Design System site by removing white background color in enabled state.
+- Updated name for "Tooltip" component in Figma library to ["Tip"](https://www.figma.com/file/Jd34cI5yyZIPb5BIwkIPU2/HPE-Tip?node-id=538%3A803) to align with the Design System site
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:58]:
```suggestion
- Updated name for "Tooltip" component in Figma library to ["Tip"](https://www.figma.com/file/Jd34cI5yyZIPb5BIwkIPU2/HPE-Tip?node-id=538%3A803) to align with the Design System site.
```

File: aries-site/src/pages/whats-new.mdx [hunk 4]
```diff
@@ -51,9 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## July 26 - August 31, 2021 
+- Published the [Notification](https://design-system.hpe.design/components#visualizations) and [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications) & [Status Indicator](https://design-system.hpe.design/templates/status-indicator) content with examples. 
+- Updated [toolbar button to include an icon only variant + changed stroke from 2px to 1px](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1) 
+- Updated [Default Button](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=681%3A5164) in Figma library to align with Design System site by removing white background color in enabled state.
+- Updated name for "Tooltip" component in Figma library to ["Tip"](https://www.figma.com/file/Jd34cI5yyZIPb5BIwkIPU2/HPE-Tip?node-id=538%3A803) to align with the Design System site 
+- Added [numerical values to both sides of the RangeInput component](https://www.figma.com/file/BqCjvjc0rECQ4Ln2QhyjNi/HPE-Range-Input-Component?node-id=0%3A1)
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:59]: hyperlink should be on the subject of the sentence, in this case, "numerical values" --> ideally this hyperlinks [directly to an example](https://www.figma.com/file/BqCjvjc0rECQ4Ln2QhyjNi/HPE-Range-Input-Component?node-id=9%3A66) of the subject.

```suggestion
- Added [numerical values](https://www.figma.com/file/BqCjvjc0rECQ4Ln2QhyjNi/HPE-Range-Input-Component?node-id=0%3A1) labeling each end of RangeInput.
```

File: aries-site/src/pages/whats-new.mdx [hunk 5]
```diff
@@ -51,9 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- Updated [toolbar button to include an icon only variant + changed stroke from 2px to 1px](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1) 
+- Updated [Default Button](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=681%3A5164) in Figma library to align with Design System site by removing white background color in enabled state.
+- Updated name for "Tooltip" component in Figma library to ["Tip"](https://www.figma.com/file/Jd34cI5yyZIPb5BIwkIPU2/HPE-Tip?node-id=538%3A803) to align with the Design System site 
+- Added [numerical values to both sides of the RangeInput component](https://www.figma.com/file/BqCjvjc0rECQ4Ln2QhyjNi/HPE-Range-Input-Component?node-id=0%3A1) 
+- Cleaned up Ordering of Items Figma file. The actions [item is now the new toolbar button which was previously default button.](https://www.figma.com/file/TonE8uQ33e5QJRPknZx36q/Ordering-Of-Items)
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:60]: Can we deeplink this directly to one of the actions menus? File is large and would be more helpful for the link to target the subject.

Also, I think we are misusing "Button" here. "Actions" is acting as a Menu component here, yet is built with a Button component. I'd like for us to get aligned on the usage.

```suggestion
- Cleaned up Ordering of Items Figma file. The [actions menu](https://www.figma.com/file/TonE8uQ33e5QJRPknZx36q/Ordering-Of-Items) is now styled similarly to the new toolbar button.
```

File: aries-site/src/pages/whats-new.mdx [hunk 6]
```diff
@@ -51,9 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- Updated [Default Button](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=681%3A5164) in Figma library to align with Design System site by removing white background color in enabled state.
+- Updated name for "Tooltip" component in Figma library to ["Tip"](https://www.figma.com/file/Jd34cI5yyZIPb5BIwkIPU2/HPE-Tip?node-id=538%3A803) to align with the Design System site 
+- Added [numerical values to both sides of the RangeInput component](https://www.figma.com/file/BqCjvjc0rECQ4Ln2QhyjNi/HPE-Range-Input-Component?node-id=0%3A1) 
+- Cleaned up Ordering of Items Figma file. The actions [item is now the new toolbar button which was previously default button.](https://www.figma.com/file/TonE8uQ33e5QJRPknZx36q/Ordering-Of-Items) 
+
 ## July 19-23, 2021
 - Iterated on Notifications taxonomy based on research results.
-- Continued [ideating on written guidance for Navigation] (https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A20).
+- Continued [ideating on written guidance for Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A20).
```
Comment (halocline) [inline aries-site/src/pages/whats-new.mdx:64]: object of the sentence can be used as secondary choice of hyperlink as "continued ideating" (subject) doesn't have a great place to link to.

```suggestion
- Continued ideating on [written guidance for Navigation](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1%3A20).
```

**Review by halocline — COMMENTED** [review]
A few little cleanups.

A few soft opinions on hyperlinks for us to consider adopting as a group.

-- Need to align on whether "Actions" is a Menu or a Button in function. Does not need to interfere with this PR.

**Comment by halocline** [comment]
I've accepted the AppliTools snapshot changes due to new calendar month. Left the Firefox displacement unmerged for the time being.

**Review by halocline — APPROVED** [review]

**Pull request merged**